var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -54,6 +54,8 @@ class IntelMkl(IntelPackage):
     version('11.3.2.181', sha256='bac04a07a1fe2ae4996a67d1439ee90c54f31305e8663d1ccfce043bed84fc27',
             url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/8711/l_mkl_11.3.2.181.tgz')
 
+    depends_on('cpio')
```

Comment:
type='build'

Reply:
Done, thanks for the review @adamjstewart.